src/Kunstmaan/TranslatorBundle/Command/ExportTranslationsCommand.php

```diff
@@ -3,14 +3,39 @@
 namespace Kunstmaan\TranslatorBundle\Command;
 
 use Kunstmaan\TranslatorBundle\Model\Export\ExportCommand;
+use Kunstmaan\TranslatorBundle\Service\Command\Exporter\ExportCommandHandler;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Serializer\Exception\InvalidArgumentException;
 
+/**
+ * @final since 5.1
+ * NEXT_MAJOR extend from `Command` and remove `$this->getContainer` usages
+ */
 class ExportTranslationsCommand extends ContainerAwareCommand
 {
+    /**
+     * @var ExportCommandHandler
+     */
+    private $exportCommandHandler;
+
+    public function __construct($translationRepository = null)
+    {
+        parent::__construct();
+
+        if (!$translationRepository instanceof ExportCommandHandler) {
+            @trigger_error(sprintf('Passing a command name as the first argument of "%s" is deprecated since version symfony 3.4 and will be removed in symfony 4.0. If the command was registered by convention, make it a service instead. ', __METHOD__), E_USER_DEPRECATED);
+
+            $this->setName(null === $translationRepository ? 'kuma:translator:export' : $translationRepository);
```

Comment:
Can be changed to $this->setName($translationRepository ?? 'kuma:translator:export');

Reply:
Sf 3.4 and the current 5.0 branch (https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/5.0/composer.json#L19) still allow php 5.6. Should we bump this requirement or keep it 5.6 compatible for now?